breba_app/builder_agent/agent.py

```diff
@@ -106,7 +106,10 @@ def is_diff_present(self, state: State) -> bool:
 
         if len(state["messages"]) > 1:
             message = state["messages"][-1].content
-            logger.info(f"Verifying if diff message: {message}")
+            clean_preview = " ".join(message.split())
```

Comment:
In these cases, I typically need the entire message. I use it to replicate an issue. If the message is truncated, it is harder to verify what actually happened and repeat it. Since we don't have transaction_ids and json logging (we want it), I have to print all the important information to be able to see the state of the transaction at any given point.